pages/index.js

```diff
@@ -1,23 +1,35 @@
+import { useState } from "react";
 import Heading from "@/components/Heading";
 import EmotionForm from "@/components/EmotionForm";
 import EmotionList from "@/components/EmotionList";
-import Image from "next/image";
 import styled from "styled-components";
-import SliderIntensity from "@/components/SliderIntensity";
+import SearchBar from "@/components/Searcbar/Searchbar";
```

Comment:
As already pointed out in the comment to the file for the component itself: you might want to change the import statement, if you change the corresponding file.